WordPress/src/main/java/org/wordpress/android/ui/media/MediaSettingsActivity.java

```diff
@@ -991,6 +996,7 @@ private void saveMediaToDevice() {
         }
         request.allowScanningByMediaScanner();
         request.setNotificationVisibility(DownloadManager.Request.VISIBILITY_VISIBLE);
+        request.addRequestHeader("User-Agent", WordPress.getDefaultUserAgent());
```

Comment:
I think we should use `WordPress.getUserAgent()` instead. This adds application information as [explained here](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/WordPress.java#L604-L627).
Checked the backend and effectively that's the common way we're using it to detect it's one of our apps back there.

Reply:
Oh right! I missed that method. Updated the PR.